components/Cta/CtaLink.tsx

```diff
@@ -32,10 +33,14 @@ export const CtaLink = React.forwardRef<HTMLAnchorElement, CtaLinkProps>(
       email,
       target,
       anchor,
+      fieldtype, // Extracted to prevent it from being included in sbLinkProps
       // External link in Storyblok can have additional custom attributes
       ...sbLinkProps
     } = sbLink || {};
 
+    // Keep only props with non empty values from sbLinkProps
+    const nonEmptySbLinkProps = Object.fromEntries(Object.entries(sbLinkProps).filter(([_, value]) => value !== '' && value !== null && value !== undefined));
```

Comment:
You need to filter out these vars because they are sent from SB correct?

Reply:
Yes, I found out that if the user expand that accordion to add more custom attributes, even if they don't add any values to `rel` and `title` (these two are always there in the UI), rel="" and title="" get passed into the data.